apps/test/unit/code-studio/pd/form_components/FormControllerTest.js

```diff
@@ -138,29 +138,30 @@ describe('FormController', () => {
 
       describe('Submitting', () => {
         let server;
-        let submitButton;
+
+        const submitButton = () =>
+          form.find('button').filterWhere(button => button.text() === 'Submit');
```

Comment:
Changed `submitButton` into a helper method because [element referential integrity is no longer
preserved](https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#element-referential-identity-is-no-longer-preserved) in enzyme 3, so we have to retrieve this element again our simulate call.